src/components/CreateAnalysis/CreateAnalysisDetail.tsx

```diff
@@ -62,9 +62,9 @@ const CreateAnalysisDetail: React.FC<CreateAnalysisDetailProps> = ({setIsExpande
                   <h3>Patient Gender</h3>
                 </div>
                 <div className="detail-patient-name-age-info">
-                  <p> {CreateAnalysisService.calculatePatientAge(patientBirthdate)}y </p>
+                  <p> {Utils.calculatePatientAge(patientBirthdate)}y </p>
                   <p> {patientBirthdate} </p>
```

Comment:
I have noticed that patient birth date format is "YYYY-MM-DD", which is inconsistent with the format defined in `Utils.formatDate()`

I would suggest to either

a) Make patient birth date format be "YYYY MM DD"
```suggestion
                  <p> {Utils.formatDate(patientBirthdate)} </p>
```

OR

b) Update the format inside `Utils.formatDate()` to "YYYY-MM-DD"